Cargo.toml

```diff
@@ -28,6 +28,8 @@ mysql_common = "0.22"
 byteorder = "1"
 chrono = "0.4"
 time = "0.2.25"
+rustls = "0.20.0-beta2"
```

Comment:
Hmm, why do we need the beta here?

Reply:
I need `0.20.0` because I can't see how to achieve what I want in earlier versions.  Even if it's somehow possible, I'd like to avoid the rework.    If you are happy with it, the best approach might be to hold off merging this until `0.20.0` is released, which doesn't seem far off. See https://github.com/rustls/rustls/issues/813

Reply:
I bumped to 0.20.0 final, so I think this PR is ready now @jonhoo